packages/backend/apps/multitenancy/tests/test_middleware.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+from graphql_relay import to_global_id
+from unittest.mock import Mock
+
+from ..middleware import get_current_tenant, get_current_user_role
+
+
+pytestmark = pytest.mark.django_db
+
+
+class TestTenantUserRoleMiddlewareGetCurrentTenant:
+    def test_get_current_tenant_with_tenant_id(self, tenant_factory):
+        tenant = tenant_factory(name="Test Tenant")
```

Comment:
Not sure if this test is good tbh. If `get_current_tenant` would return first/last row in DB it will also pass it... on the other hand there is a test below which checks if it will return `None` for non existing ID. WDYT?